charts/kong/CHANGELOG.md

```diff
@@ -13,6 +13,10 @@
 
 * The target port for cmetrics should only be applied if the ingress controller is enabled.
   [#926](https://github.com/Kong/charts/pull/926)
+* Fix RBAC for Gateway API v1.
+  [#928](https://github.com/Kong/charts/pull/928)
```

Comment:
I think the two items could be written in one entry.

Reply:
This is to prevent the need for us to backport a certain commit to other branches, so it has been split. If necessary, I can merge them.

Reply:
I believe those are OK as is. They cover different subjects, different fixes.